test/conformance/suite_test.go

```diff
@@ -181,3 +237,7 @@ func exitOnErr(err error) {
 		os.Exit(1)
 	}
 }
+
+func expressionRoutesEnabled() bool {
+	return strings.ToLower(expressionRoutesMode) == "true"
+}
```

Comment:
Ditto about env but this one is yet undefined in `testenv`. We could add it in https://github.com/Kong/kubernetes-ingress-controller/blob/665a8e83deb3e1e4e4f41d4a26d6a0635fc6b0b5/test/internal/testenv/testenv.go to everyone to reuse and give it a proper godoc.

Reply:
Moved, I have an idea to improve code in `testenv` a little bit, I'll add it to the separate PR mentioned before